fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/GlobalConfigurationProperty.java

```diff
@@ -114,8 +114,8 @@ public Date getDateValue(){
 
         final String passwordPropertyName = "force-password-reset-days";
         if (this.name.equalsIgnoreCase(passwordPropertyName)) {
-            if (this.enabled == true && command.hasParameter(valueParamName) && this.value == 0 || this.enabled == true
-                    && !command.hasParameter(valueParamName) && previousValue == 0) { throw new ForcePasswordResetException(); }
+            if (((this.enabled == true) && command.hasParameter(valueParamName) && (this.value == 0)) || ((this.enabled == true)
+                    && !command.hasParameter(valueParamName) && (previousValue == 0))) { throw new ForcePasswordResetException(); }
```

Comment:
Wait, this looks weird to me, shouldn't this just be:

```suggestion
            if ((this.enabled == true && command.hasParameter(valueParamName) && this.value == 0) || (this.enabled == true
                    && !command.hasParameter(valueParamName) && previousValue == 0)) { throw new ForcePasswordResetException(); }
```

Reply:
yeah will modify this